app/controllers/idv/inherited_proofing_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Idv
   class InheritedProofingController < ApplicationController
+    before_action :confirm_two_factor_authenticated
+
     include Flow::FlowStateMachine
     include IdvSession
```

Comment:
We may be able to resolve this issue by reordering the `include`s instead, since [`IdvSession` is already checking that the user is authenticated](https://github.com/18F/identity-idp/blob/3e90edd8331087e985872f9031c5bd2ab49dbc6a/app/controllers/concerns/idv_session.rb#L6), but it's not being reached since `FlowStateMachine` is where the error is happening.

```suggestion
    include IdvSession
    include Flow::FlowStateMachine
```